website/public/js/controllers/authCtrl.js

```diff
@@ -115,5 +122,14 @@ angular.module('habitrpg')
           alert("Signin error: " + e.error.message );
         });
       }
+
+      //reCaptcha
+      this.publicKey = "6LfGQ8wSAAAAAHP86aI1YMfRq3Lg1lxGo-HYxO7u";
```

Comment:
This should be a config variable. It's not a security concern that it's in the repo, since this is a public key, but it should be a config variable nonetheless.